docs/src/content/en/examples/rag/usage/graph-rag.mdx

```diff
@@ -129,7 +128,7 @@ Generate embeddings for the chunks and store them in the vector database:
 
 ```typescript copy showLineNumbers{56} title="index.ts"
 const { embeddings } = await embedMany({
-  model: openai.embedding("text-embedding-3-small"),
+  model: "openai/text-embedding-3-small",
```

Comment:
I _think_ you need to wrap the model string in ⁠ModelRouterEmbeddingModel, as shown here: https://mastra.ai/models/embeddings#quick-start

that sounds annoying to update in all these files - you can probably skip ⁠examples/rag/* since we’re overhauling that later, but it’s worth double‑checking the docs. In other words, it's okay not to update examples/rag/* in this PR